docs/general/web3-and-polkadot.md

```diff
@@ -0,0 +1,196 @@
+nodes and secure the network. Strong incentives are possible because blockchain is a trustless
+system where there are no intermediaries between who sends a message and who receives it. Such
+incentives, coupled with punishment for bad behavior, ensure that most of the participants make the
+interest of the network and work together to improve it.
+
+But from where are those incentives coming from? Polkadot's native token
+[DOT is inflationary](../learn/learn-inflation.md). Inflation is used to pay validators for running
+nodes and reward nominators for providing the necessary stake to secure the network. Depending on
+the staking rate, part of the inflation is diverted to the treasury.
+
+### Governance and Treasury
+
+In {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} an on-chain
+[treasury](../learn/learn-polkadot-opengov-treasury.md) together with an
+[open governance](../learn/learn-polkadot-opengov.md) model allow to access funds in a fully
+decentralized manner without any bank transaction whatsoever. This opens up the possibility to come
+to a decision through on-chain voting mechanism, promoting a sense of community and creating an
+independent socio-economical environment.
+
+## Decentralized Access Points
+
+But what if we have data we own stored on a resilient blockchain, but the only way to access the
+blockchain is through an RPC server? Whoever is behind the server or an attacker could present us
+data that is not the truth. We trust the server provider that the data we see is the truth. How can
+we verify that the data are true without trusting anybody?
+
+Here is where light clients play a key role. Light clients are clients that can sit on a web browser
+and can fetch data directly from blockchain full nodes and verify such data using other nodes.
+
+The figure below shows the architectural difference between web2 and web3 applications.
+
+![light-clients](../assets/light-clients.png)
+
+In web2 applications, data are stored on a centralized server, while in web3 applications, data (or
+better data proofs) are stored on the blockchain. With light clients, we can access blockchain data
+through a full node and verify the validity of such data by synching to other nodes. In this way, we
+can always verify that the data we see is the truth, which is done automatically by the light
+client. Polkadot has a browser-embedded light client
+[Substrate connect](../build/build-substrate.md) that uses the
+[smoldot](https://github.com/smol-dot/smoldot) codebase. Most web3 applications today access
+blockchain data through a centralized RPC server.
+
+## Interoperability
+
+But what if we create our identity under one blockchain, one specific consensus? Wouldn't it be
+helpful to use that identity under different consensuses? Transferring information and economic
+value is key to achieving the web3 vision of having a collaborative and trustless internet. Polkadot
```

Comment:
Cross chain interoperability of accounts is only one use case of iteroperability, but not a foundational argument for interoperability. This section can benefit from a better start. A summary of this article would be nice to have here https://chain.link/education-hub/blockchain-interoperability

```suggestion
Transferring information and economic
value is key to achieving the web3 vision of having a collaborative and trustless internet. Polkadot
```